src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -457,7 +457,15 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                 }
             };
 
-            BreakChangeEvent += _breakpointManager.BreakpointModified;
+            BreakChangeEvent += async delegate (object o, EventArgs args)
+            {
+                try
+                {
+                    await _breakpointManager.BreakpointModified(o, args);
+                }
+                catch (Exception)
+                { }
```

Comment:
>{ [](start = 16, length = 1)

Call `when (ExceptionHelper.BeforeCatch(e, Logger, reportOnlyCorrupting: true))`?

Reply:
I can add that to this section and the other try/catch above too.